crates/miden-objects/src/account/delta/storage.rs

```diff
@@ -49,90 +39,129 @@ impl AccountStorageDelta {
     /// Returns an error if:
     /// - Any of the updated slot is referenced from both maps, which means a slot is treated as
     ///   both a value and a map slot.
-    pub fn from_parts(
-        values: BTreeMap<StorageSlotName, Word>,
-        maps: BTreeMap<StorageSlotName, StorageMapDelta>,
-    ) -> Result<Self, AccountDeltaError> {
-        let delta = Self { values, maps };
-        delta.validate()?;
+    pub fn from_raw(deltas: BTreeMap<StorageSlotName, StorageSlotDelta>) -> Self {
+        Self { deltas }
+    }
```

Comment:
We are no longer returning errors from here - so, doc comments need to be updated.